<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BF4</class>
 <widget class="QTabWidget" name="BF4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="tab_pl">
   <attribute name="title">
    <string>Players</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget_pl_playerList">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Kills</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deaths</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Score</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ping</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GUID</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_ml">
   <attribute name="title">
    <string>Maplist</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="3">
     <widget class="QLabel" name="label_ml_current">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>235</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" rowspan="3">
     <widget class="QListWidget" name="listWidget_ml_avaliable"/>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_ml_add_2">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ml_add">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ml_avaliable">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gamemode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_ml_gameMode"/>
      </item>
     </layout>
    </item>
    <item row="2" column="3" rowspan="4">
     <widget class="QListWidget" name="listWidget_ml_current"/>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_ch">
   <attribute name="title">
    <string>Chat</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_ch_duration"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_ch_input"/>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit_ch_output">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_ch_send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_ch_mode"/>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="comboBox_ch_target"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_ev">
   <attribute name="title">
    <string>Events</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit_ev_output">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_co">
   <attribute name="title">
    <string>Console</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_co_input"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_co_send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_co_output">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>treeWidget_pl_playerList</tabstop>
  <tabstop>lineEdit_co_input</tabstop>
  <tabstop>pushButton_co_send</tabstop>
  <tabstop>textEdit_co_output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
